test/model_recover.cc

```diff
@@ -96,7 +96,7 @@ int main(int argc, char *argv[]) {
   std::string name = rabit::GetProcessorName();
 
   int max_rank = rank;
-  rabit::Allreduce<op::Max>(&max_rank, sizeof(int));
+  rabit::Allreduce<op::Max>(&max_rank, 1);
```

Comment:
Why is this 1?

Reply:
this is count, so we are syncing one int